Application/EdFi.Ods.AdminApi/Infrastructure/Services/ClaimSetEditor/GetResourcesByClaimSetIdQueryV53Service.cs

```diff
@@ -1,242 +1,253 @@
-                    actions = AddStrategyToChildResource(createDefaultStrategy, Action.Create);
-
-                    var readDefaultStrategy = defaultAuthStrategiesForChildren.SingleOrDefault(x =>
-                        x.ResourceClaim.ResourceClaimId == resourceClaim.ResourceClaimId &&
-                        x.Action.ActionName == Action.Read.Value)?.AuthorizationStrategy;
-                    actions = AddStrategyToChildResource(readDefaultStrategy, Action.Read);
-
-                    var updateDefaultStrategy = defaultAuthStrategiesForChildren.SingleOrDefault(x =>
-                        x.ResourceClaim.ResourceClaimId == resourceClaim.ResourceClaimId &&
-                        x.Action.ActionName == Action.Update.Value)?.AuthorizationStrategy;
-                    actions = AddStrategyToChildResource(updateDefaultStrategy, Action.Update);
-
-                    var deleteDefaultStrategy = defaultAuthStrategiesForChildren.SingleOrDefault(x =>
-                        x.ResourceClaim.ResourceClaimId == resourceClaim.ResourceClaimId &&
-                        x.Action.ActionName == Action.Delete.Value)?.AuthorizationStrategy;
-                    actions = AddStrategyToChildResource(deleteDefaultStrategy, Action.Delete);
-
+                }
+                else
+                {
+                    var createDefaultStrategy = defaultAuthStrategiesForChildren.SingleOrDefault(x =>
+                        x.ResourceClaim.ResourceClaimId == resourceClaim.ResourceClaimId &&
+                        x.Action.ActionName == Action.Create.Value)?.AuthorizationStrategy;
+                    actions = AddStrategyToChildResource(createDefaultStrategy, Action.Create);
+
+                    var readDefaultStrategy = defaultAuthStrategiesForChildren.SingleOrDefault(x =>
+                        x.ResourceClaim.ResourceClaimId == resourceClaim.ResourceClaimId &&
+                        x.Action.ActionName == Action.Read.Value)?.AuthorizationStrategy;
+                    actions = AddStrategyToChildResource(readDefaultStrategy, Action.Read);
+
+                    var updateDefaultStrategy = defaultAuthStrategiesForChildren.SingleOrDefault(x =>
+                        x.ResourceClaim.ResourceClaimId == resourceClaim.ResourceClaimId &&
+                        x.Action.ActionName == Action.Update.Value)?.AuthorizationStrategy;
+                    actions = AddStrategyToChildResource(updateDefaultStrategy, Action.Update);
+
+                    var deleteDefaultStrategy = defaultAuthStrategiesForChildren.SingleOrDefault(x =>
+                        x.ResourceClaim.ResourceClaimId == resourceClaim.ResourceClaimId &&
+                        x.Action.ActionName == Action.Delete.Value)?.AuthorizationStrategy;
+                    actions = AddStrategyToChildResource(deleteDefaultStrategy, Action.Delete);
+
+                                        var readChangesDefaultStrategy = defaultAuthStrategiesForChildren.SingleOrDefault(x =>

```

Comment:
Here again we have the addition of `readChanges`. I copied these directly from the same v6 service file.